python/lsst/ts/salobj/topics/controller_command.py

```diff
@@ -141,6 +167,16 @@ def _queue_one_item(self, data):
                 ack = self.salinfo.makeAckCmd(private_seqNum=data.private_seqNum,
                                               ack=sal_enums.SalRetCode.CMD_COMPLETE, result="Done")
             self.ack(data, ack)
+        except asyncio.CancelledError:
+            ack = self.salinfo.makeAckCmd(private_seqNum=data.private_seqNum,
+                                          ack=sal_enums.SalRetCode.CMD_ABORTED, error=1,
+                                          result=f"Aborted", truncate_result=True)
+            self.ack(data, ack)
+        except asyncio.TimeoutError:
+            ack = self.salinfo.makeAckCmd(private_seqNum=data.private_seqNum,
+                                          ack=sal_enums.SalRetCode.CMD_TIMEOUT, error=1,
+                                          result=f"Aborted", truncate_result=True)
```

Comment:
Should the `result` for a timeout be `Timed out` (or something like that) instead of `Aborted`?